webshop-backend/src/shopping/ShoppingCart.java

```diff
@@ -0,0 +1,24 @@
+package shopping;
+
+import java.util.ArrayList;
+
+public class ShoppingCart {
+    private ArrayList<Product> products;
+    private int totalPrice;
+    private User owner;
+
+    public ShoppingCart(ArrayList<Product> products, int totalPrice, User owner) {
+        this.products = products;
+        this.totalPrice = totalPrice;
+        this.owner = owner;
+    }
+
+    public void addProduct(Product p) {
+        products.add(p);
+    }
```

Comment:
Since the `Product` class is not implemented yet, you couldn't implement the required behavior yet:
![image](https://github.com/deb05121/webshop-backend/assets/42215450/3f144fd6-fcc1-4009-84a6-cd0e86120fae)
I believe there are 2 options
- Wait until someone implements the `Product` class
- Implement a dummy `Product` class with a `stock` field

What do you think?